sourced/ml/mining/cmd/clickhouse2deps.yaml

```diff
@@ -0,0 +1,42 @@
+cpp:
+  filters:
+    - - "uptypes = ['uast:InlineImport']"
```

Comment:
`- -` :thinking: ?

Reply:
list of list, in the jinja we join conditions in the same list with `AND`, and then those are joind with `OR`. but I actually made a mistake